code/modules/atmospherics/machinery/components/gas_recipe_machines/atmos_machines_recipes.dm

```diff
@@ -148,3 +148,76 @@ GLOBAL_LIST_INIT(gas_recipe_meta, gas_recipes_list())
 	energy_release = 29000
 	requirements = list(/datum/gas/antinoblium = 5, /datum/gas/zauker = 20, /datum/gas/bz = 7.5)
 	products = list(/obj/item/stack/sheet/mineral/zaukerite = 2)
+
+/datum/gas_recipe/hypertorus_fusion_reactor
+	machine_type = "Hypertorus_Fusion_Reactor"
```

Comment:
Why do this as a recipe? you're just ending up redefining the meaning of these base vars.

Reply:
I could just make a datum/hfr_fuel_mix
Would that work?